### PR TITLE
Lambda details

### PR DESCRIPTION
**Pull request by carlgarner**
Tidy up and standardise spacing between things
Add aws-regions.txt as it appears to be the same regardless of parameters passed (as suggested in doc)
Update README
Add flag to output number of versions and name of lambda being worked on

### CODE REVIEW

**Comment by karl-cardenas-coding** [comment]
Hi @carlgarner, thanks for the PR. I'm gonna review these changes later today when I get a moment. Two things that stand out. The readme update, we need to have that file generated locally to compile. That's how we add support for new AWS regions, by querying the EC2 API and getting the list during compile. 

The other question I have is about IBytes. What is the benefit of switching over to IBytes?

**Comment by carlgarner** [comment]
Fair point on the regions, though I don't believe they change that frequently. You _could_ keep this stored to simplify the build process, but equally, is it really needed? A user would know what region they're operating in and this would either be part of their profile (if they've chosen to set a default) or something they know and are happy passing at run-time.

As to the IBytes, Bytes uses the 1000 multiplier and IBytes is 1024. I would argue that most are used to seeing things in the 1024 format, however, I'd changed this because I believe there may be a calculation error somewhere. When running the app against the account I'm working with, the Lambda usage is ~69Gb, yet the dryrun report was claiming a value ~77Gb. I hadn't got to the bottom of this when I made the PR which was meant to be, largely, for the version count and lambda details.

Unfortunately, I haven't got much further with it. This is a first foray into GoLang as well.

File: cmd/clean.go [hunk 1]
```diff
@@ -226,16 +227,13 @@ func executeClean(region string) error {
 			numVerDeleted := countDeleteVersions(globalLambdaDeleteInputStructs)
 			log.Info(fmt.Sprintf("%d unique versions will be removed in an actual execution.", numVerDeleted))
 			spaceRemovedPreview := calculateSpaceRemoval(globalLambdaDeleteList)
-			log.Info(fmt.Sprintf("%s of storage space will be removed in an actual execution.", humanize.Bytes(uint64(spaceRemovedPreview))))
-
-		}
-
-		if !DryRun {
+			log.Info(fmt.Sprintf("%s of storage space will be removed in an actual execution.", humanize.IBytes(uint64(spaceRemovedPreview))))
+		} else {
```
Comment (karl-cardenas-coding) [inline cmd/clean.go:231]: We don't really need the else. Let's just end the original `if DryRun {}` block and allow the rest of the code to execute. That's cleaner.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
I left some comments below. The main changes are related to naming. We also need to update the README to include information about the flag. 

Also, this project uses semantic release, so in your next commit, use `"fix: <yourCommentHere>" `  otherwise, the CI will not publish the next release.

File: README.md [hunk 2]
```diff
@@ -176,10 +176,9 @@ $ glc clean -r us-west-2
 2021/03/04 20:42:46 ............
 ```
 ## Compile
-If you want to complile the binary, clone the project to your local system. Ensure you have `Go 1.17` installed. This tool leverages the Golang [embed](https://golang.org/pkg/embed/) functionality. A file named `aws-regions.txt` is expected in the `cmd/` directory.  You need valid AWS credentials in order to generate the file.
+If you want to complile the binary, clone the project to your local system. Ensure you have `Go 1.18` installed. 
 ```shell
```
[the remark this answers is not in the document]
Comment (carlgarner) [reply README.md]: Kept Go 1.18, but reverted the rest

File: cmd/root.go [hunk 3]
```diff
@@ -25,6 +25,8 @@ var (
 	DryRun bool
 	// LambdaListFile points a file that contains a listof Lambdas to delete
 	LambdaListFile string
+	// MoreDetail is to show information about the Lambda being worked on
+	MoreDetail bool
```
[the remark this answers is not in the document]
Comment (carlgarner) [reply cmd/root.go]: Expanded to MoreLambdaDetail

File: cmd/root.go [hunk 4]
```diff
@@ -55,6 +57,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&RegionFlag, "region", "r", "", "Specify the desired AWS region to target.")
 	rootCmd.PersistentFlags().StringVarP(&ProfileFlag, "profile", "p", "", "Specify the AWS profile to leverage for authentication.")
 	rootCmd.PersistentFlags().StringVarP(&LambdaListFile, "listFile", "l", "", "Specify a file containing Lambdas to delete.")
+	rootCmd.PersistentFlags().BoolVarP(&MoreDetail, "moreDetail", "m", false, "Set to true to show Lambda names being worked on (bool)")
```
[the remark this answers is not in the document]
Comment (carlgarner) [reply cmd/root.go]: Updated the detail for the flag as well

File: cmd/clean.go (hunk 1 above)
Comment (carlgarner) [reply cmd/clean.go]: Modified the format of this now but had to move the duration calculation and output to a function to remove code duplication.

**Comment by karl-cardenas-coding** [comment]
@carlgarner I've been thinking about this change, and it's technically breaking change.  I'm gonna move this PR into a new branch. I want to abstract some things out to avoid changing the changes for existing users. I'll tag you when it's ready for review. Your changes will carry over to this branch.

**Pull request merged**